scripts/install.sh

```diff
@@ -542,6 +551,15 @@ function setup_config() {
         exit 1
     fi
 
+    if [[ "${INSTALL_HOSTMETRICS}" == "true" ]]; then
+        echo -e "Installing ${OS_TYPE} hostmetrics configuration"
+        HOSTMETRICS_CONFIG_URL="https://raw.githubusercontent.com/SumoLogic/sumologic-otel-collector/${CONFIG_BRANCH}/examples/conf.d/${OS_TYPE}.yaml"
+        if ! curl --retry 5 --connect-timeout 5 --max-time 30 --retry-delay 0 --retry-max-time 150 -f -s "${HOSTMETRICS_CONFIG_URL}" -o "${CONFIG_DIRECTORY}/conf.d/hostmetrics.yaml"; then
+            echo "Cannot obtain hostmetrics configuration for '${CONFIG_BRANCH}' branch"
+            exit 1
```

Comment:
I wouldn't mind putting this in a separate function, to keep the main script body cleaner.